pkg/cmd/template/schema_test.go

```diff
@@ -4,13 +4,12 @@
 package template_test
 
 import (
-	"strings"
-	"testing"
-
 	"github.com/k14s/difflib"
 	cmdtpl "github.com/k14s/ytt/pkg/cmd/template"
 	"github.com/k14s/ytt/pkg/cmd/ui"
 	"github.com/k14s/ytt/pkg/files"
+	"strings"
```

Comment:
nit: std lib imports separation

Reply:
For GoLand users:

![image](https://user-images.githubusercontent.com/12101915/106055703-9b376780-60a2-11eb-8c89-f58b41fbf54f.png)